ivy/functional/frontends/numpy/indexing_routines/inserting_data_into_arrays.py

```diff
@@ -31,6 +31,15 @@ def fill_diagonal(a, val, wrap=False):
     a = ivy.reshape(temp, shape)
 
 
+@to_ivy_arrays_and_back
+def putmask(a, mask, values, /):
```

Comment:
You could remove the `/` as it doesn't make sense because there are no kwargs after that

Reply:
Oh correct, thanks for catching this, I have updated it correctly now @bipinKrishnan